bin/dlang/make

```diff
@@ -33,22 +33,8 @@ then
     "$beaver" make d2conv
 fi
 
-# Build default target
+# Build
 "$beaver" make all
 
-# Test and upload coverage reports
-test_cov()
-{
-    what="$1"
-    "$beaver" make "$what"
-    "$beaver" dlang codecov -n "$what" -F "$what"
-    rm -f .*.lst
-}
-
-test_cov unittest
-
-# XXX: Disabled for now because only the last integration test results will be
-#      uploaded until MakD can make sure integrationtest reports are stored
-#      separately. See: https://github.com/sociomantic-tsunami/makd/issues/107
-#test_cov integrationtest
-"$beaver" make integrationtest
+# Test
+"$beaver" make test
```

Comment:
Here is the change that needed the adaptation of the tests: `make unittest; make integration test` -> `make test`.